ComponentKit/Core/Scope/CKScopedComponent.h

```diff
@@ -0,0 +1,13 @@
+//
+//  CKScopedComponent.h
+//  ComponentKit
+//
+//  Created by Oliver Rickard on 3/27/17.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+@protocol CKScopedComponent <NSObject>
+
+@end
```

Comment:
I was surprised that the scope infra actually doesn't care about the component type at all, thus this protocol is empty. I still want to keep it so we get type annotations on the id types for which are the components, and which are the controllers in the interfaces of the scope infra. Straight id's are unreadable.